src/applications/static-pages/homepage-hero-randomizer/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import './sass/homepage-hero.scss';
+
+// shuffle array using Fisher-Yates shuffle
+const randomizeOrder = imageArray => {
+  const tempArray = imageArray;
+  let currentIndex = Object.keys(tempArray).length;
+  let randomIndex;
+
+  // While there remain elements to shuffle...
+  while (currentIndex !== 0) {
+    // Pick a remaining element...
+    randomIndex = Math.floor(Math.random() * currentIndex);
+    currentIndex--;
+
+    // And swap it with the current element.
+    [tempArray[currentIndex], tempArray[randomIndex]] = [
+      tempArray[randomIndex],
+      tempArray[currentIndex],
+    ];
+  }
+
+  // grab first 3 items
+  return tempArray.slice(0, 3);
+};
+
+const HeroRandom = () => {
+  const imagePaths = [
+    { id: 1, name: 'babu' },
+    { id: 2, name: 'britton' },
+    { id: 3, name: 'clark' },
+    { id: 4, name: 'cogswell' },
+    { id: 5, name: 'dinh' },
+    { id: 6, name: 'henline' },
+    { id: 7, name: 'pickard-locke' },
+    { id: 8, name: 'singleton' },
+  ];
+
+  const gradientValues = ['#105f9f', '#144e84', '#143e6a'];
```

Comment:
I think this should likely be done in scss.  This brings to light the idea that we'll likely need to add these colors to the approved VA design system.  This might take some extra work/governance.

Reply:
As discussed in scrum 4/21, we will keep it simple for this iteration, then refine later.